auth_jwt.go

```diff
@@ -339,6 +343,11 @@ func (mw *GinJWTMiddleware) MiddlewareInit() error {
 		mw.CookieName = "jwt"
 	}
 
+	// bypass other key settings if KeyFunc is set
+	if mw.KeyFunc != nil {
```

Comment:
Why need to return nil?

Reply:
I guess this is because "bypass other key settings if KeyFunc is set".

Reply:
Yes .. been a while since I wrote this, but I think its because the code below that point is not relevant if using the KeyFunc (and returning nil because there's no error).